package.json

```
@@ -17,11 +17,10 @@
     "prepublishOnly": "npm run build",
     "prebuild": "rimraf dist",
     "build": "babel src --out-dir dist --copy-files --ignore **/*.test.js",
-    "lint": "xo",
+    "lint": "eslint . --ext js,ts --cache",
     "postbuild": "rimraf dist/**/*.test.js",
-    "test:watch": "jest --watch",
-    "test:xo": "xo && jest --coverage",
-    "test": "jest"
+    "test": "npm-run-all lint test:*",
+    "test:jest": "jest"
```

I think we don't need npm-run-all.

```suggestion
    "test": "yarn lint && yarn test:jest",
    "test:jest": "jest"
```

---

You only need `npm-run-all` if you have multiple things you want to run, and specifically use glob-like patterns to run them. Also, it parallelizes things, and I don't think we need that here.